src/subdomains/generic/kyc/services/name-check.service.ts

```diff
@@ -101,6 +111,23 @@ export class NameCheckService implements OnModuleInit {
     return RiskStatus.SANCTIONED;
   }
 
+  async getRiskDataAndUploadPdf(userData: UserData, name: string, dob?: Date): Promise<DilisenseApiData> {
+    const { data: riskData, pdfData } = await this.dilisenseService.getRiskData(name, dob);
+
+    // upload file
+    const { contentType, buffer } = Util.fromBase64(pdfData);
+    await this.documentService.uploadFile(
+      userData,
+      FileType.NAME_CHECK,
+      `nameCheck-${Date.now().toLocaleString()}`,
+      buffer,
```

Comment:
We normally have the date at the beginning of the name, and in short format 🤔  e.g. like this: `20250402-onboarding-0-311135-211146.pdf`